controllers/configmap_controller.go

```diff
@@ -304,7 +309,8 @@ func (r *ConfigMapReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			return r.isValidConfigMap(e.Object)
 		},
 		UpdateFunc: func(e event.UpdateEvent) bool {
-			return r.isValidConfigMap(e.ObjectNew)
+			return r.isValidConfigMap(e.ObjectNew) ||
+				r.isKubeAPIServerServingCAConfigMap(e.ObjectNew)
```

Comment:
Is not that simple, it also [accounts](https://github.com/openshift/windows-machine-config-operator/blob/3a2c73a59041d7be5682e6af68cd908a1161ba45/controllers/configmap_controller.go#L326) for the `windows-services` ConfigMap.

Reply:
Because: 
- need to react only to `Update` events
- the KubeAPIServer CA ConfigMap is a resource outside of the watched namespace (`r.watchNamespace`)

Reply:
I suggest isValidConfigMap --> isWatchNamespaceConfigMap

Reply:
Agree 💯 , but I'm skeptical to bring more unrelated changes into this PR.

Reply:
I am fine with keeping the name as is it:  Just don't want to have too many changes that are not related to actual change.
I already see filename exist changes that can be avoided from the previous commits.